documentation/modules/con-kafka-logging.adoc

```diff
@@ -0,0 +1,89 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+[id='con-kafka-logging-{context}']
+= Kafka loggers
+
+Kafka has its own configurable loggers:
```

Comment:
I think we need to add here the Zookeeper logger as well: see the [old docs](https://strimzi.io/docs/latest/full.html#ref-loggers-Kafka-deployment-configuration-kafka)